packages/fiori/src/UserMenu.ts

```diff
@@ -125,6 +125,18 @@ type UserMenuOtherAccountClickEventDetail = {
 	cancelable: true,
 })
 
+/**
+ * Fired when a user menu is open.
+ * @public
+ */
+@event("open")
```

Comment:
add since 2.6.0. same for close event

Reply:
show we add 2.6.0 or 2.6.0-rc.1?

Reply:
2.6.0 - since tags are pointing stable releases, not release candidates